framework/src/postprocessors/VectorPostprocessorDifferenceMeasure.C

```diff
@@ -0,0 +1,129 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "VectorPostprocessorDifferenceMeasure.h"
+#include "VectorPostprocessorInterface.h"
+
+registerMooseObject("MooseApp", VectorPostprocessorDifferenceMeasure);
+
+defineLegacyParams(VectorPostprocessorDifferenceMeasure);
```

Comment:
I don't think this is needed anymore. Can you double check with someone else?

Reply:
I don't know what `defineLegacyParams` does but when I don't include it I get a segfault.  I'll ask.